js/ui/file_uploader.js

```diff
@@ -273,6 +275,57 @@ var FileUploader = Editor.inherit({
             */
             onUploadAborted: null,
 
+
+            /**
+            * @name dxFileUploaderOptions.validation
+            * @type Object
+            * @default { allowedFileExtensions: [], maxFileSize: 0, minFileSize: 0, invalidFileExtensionMessage: "", invalidMaxFileSizeMessage: "", invalidMinFileSizeMessage: ""}
```

Comment:
Add `* @default undefined` instead of this line.

Reply:
In the menu widjet, I found a similar example with a complex property, where the default value is an object with all properties(dxMenuOptions.showFirstSubmenuMode)

Reply:
Just remove the default from here. We should do the same to the showFirstSubmenuMode and many other options that accept objects. Defaults in them [look ricidulous](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/animation/).